packages/web/src/components/listView.tsx

```diff
@@ -170,6 +208,29 @@ export function ListView<T>({
           {typeof rendered === 'string' ? <div style={{ textOverflow: 'ellipsis', overflow: 'hidden' }}>{rendered}</div> : rendered}
         </div>;
       })}
+      {contextMenu && (
```

Comment:
Showing a proper context menu is a complicated business. Perhaps we should use [popover](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using)?